src/pages/style/ModalWrapper.js

```diff
@@ -0,0 +1,8 @@
+import styled from 'styled-components';
+
+const ModalWrapper = styled.div`
```

Comment:
For your modal, you could actually use an `<aside>` - but that's a really picky comment, and up to you 😉